src/engine/renderer/Material.cpp

```diff
@@ -1887,10 +2232,47 @@ void MaterialSystem::RenderMaterial( Material& material ) {
 	}
 	material.texturesResident = true;
 
-	glMultiDrawElementsIndirect( GL_TRIANGLES, GL_UNSIGNED_INT,
-		BUFFER_OFFSET( material.staticCommandOffset * sizeof( GLIndirectBuffer::GLIndirectCommand ) ),
+	culledCommandsBuffer.BindBuffer( GL_DRAW_INDIRECT_BUFFER );
+
+	atomicCommandCountersBuffer.BindBuffer( GL_PARAMETER_BUFFER_ARB );
+
+	glMultiDrawElementsIndirectCountARB( GL_TRIANGLES, GL_UNSIGNED_INT,
+		BUFFER_OFFSET( material.surfaceCommandBatchOffset * SURFACE_COMMANDS_PER_BATCH * sizeof( GLIndirectBuffer::GLIndirectCommand )
+					   + ( culledCommandsCount * ( MAX_VIEWS * currentFrame + viewID )
+					   * sizeof( GLIndirectBuffer::GLIndirectCommand ) ) ),
+					   //+ ( culledCommandsCount * ( MAX_VIEWS * currentFrame + currentView )
+						   //* sizeof( GLIndirectBuffer::GLIndirectCommand ) ),
```

Comment:
Some space inconsistency there.

Reply:
Fixed, also removed the commented out code.